app/grandchallenge/algorithms/models.py

```diff
@@ -567,6 +570,13 @@ def get_absolute_url(self):
             kwargs={"slug": self.algorithm.slug, "pk": self.pk},
         )
 
+    @property
+    def download_url(self):
```

Comment:
This link (and the corresponding one for the model below) are not used yet, but will be once the remaining external evaluation changes are implemented, hence I added them already.